sql/core/src/test/resources/sql-tests/results/datetime.sql.out

```diff
@@ -391,3 +391,122 @@ select date '2001-10-01' - date '2001-09-28'
 struct<subtractdates(DATE '2001-10-01', DATE '2001-09-28'):interval>
 -- !query output
 3 days
+
+
+-- !query
+select to_timestamp(v, 'yyyy-MM-dd HH:mm:ss.SSSSSS[zzz]') from values
+ ('2019-10-06 10:11:12.'),
+ ('2019-10-06 10:11:12.0'),
+ ('2019-10-06 10:11:12.1'),
+ ('2019-10-06 10:11:12.12'),
+ ('2019-10-06 10:11:12.123UTC'),
+ ('2019-10-06 10:11:12.1234'),
+ ('2019-10-06 10:11:12.12345CST'),
+ ('2019-10-06 10:11:12.123456PST') t(v)
+-- !query schema
+struct<to_timestamp(t.`v`, 'yyyy-MM-dd HH:mm:ss.SSSSSS[zzz]'):timestamp>
+-- !query output
+2019-10-06 03:11:12.123
```

Comment:
The result looks weird. Where is `.123` parsed from?

Reply:
here ('2019-10-06 10:11:12.123UTC')

Reply:
This is hard to read as the output order is different from the input order. Can we break it down to multiple queries? or put the input column in the output as well, e.g. `select v, to_timestamp(v, ...`